configure

```diff
@@ -53,9 +53,27 @@ Options:
   --spidermonkey-version VSN  specify the version of SpiderMonkey to use (defaults to $SM_VSN)
   --skip-deps                 do not update erlang dependencies
   --rebar=PATH                use rebar by specified path (version >=2.6.0 && <3.0 required)
+  --tls                       enable TLS distribution and generate a certificate
```

Comment:
As is, this doesn't enable distribution? It's a nice helper function to generate an example certificate and an ssl_dist.conf file. Since we're generating it in ./dev/ we might want to rename the option as `--tls-generate-dev-cert` indicating this is just an example used for development. Another alternative, would be to just document the procedure on how to do it in  `vm.args` comments along with the rest of the comments. That way there is no ambiguity that this not an endorsement of specific expiration limits, key sizes or algorithms.

Reply:
Add comments to vm.args. Thanks for the review, @nickva